provisionerd/proto/provisionerd.proto

```diff
@@ -75,6 +75,7 @@ message CompletedJob {
         repeated provisioner.Resource resources = 2;
         repeated provisioner.Timing timings = 3;
         repeated provisioner.Module modules = 4;
+        repeated provisioner.ResourceReplacement resourceReplacements = 5;
```

Comment:
We discussed on Slack, but these changes need to bump the minor version --- unless it was already bumped since the last release, in which case you need to update the comment describing the version bump, but don't need to bump it twice in a single release.